.travis.yml

```diff
@@ -21,5 +21,4 @@ cache:
   - node_modules
 env:
   global:
-  - secure: FvNhotbMfsYm+q1UNEUX9aWYKDePjwu5uUDkd5gcsN3mvT8BmjJekDc+PvCqwaG53y7XZ1iXAmIYzm8AUPkUzum32NoRL0bLn/mAu/qXItv0nAOU4lvkCrH3uLeC5Sf3IEZh6UiZxzyPMyu0pV7yyLJLhtR1C60KX+BdpdWy8Je8307c8jdesa+VwUAJ822OvW0lfSxfAeOfMEbNaBU2VNFgQQpvzMeA2lWqWjRzVRZ2h1F+TRSKFcI4czGYyVnYBQaRzp5/44KjIU0edQ1lW5vNjqGjXp4xxCB3/GnqyNWukLvLJfoxNtN+zzCLK4mKUjK8EeBX9zWxGTl2YPtNG5CrsX7lBHJQ52/XEc9oV9wcNodn4KlG/bsVwRSwiLIyFewSBY3h+Aqvv/RnANVDeEWrhB1NVIIGyiRQFpDUjoxceI7Mm7jSIYN6R1TOvwG/7FW7CnwGb9I1v4Ra2ixWg4e/xULSEboZZkLvCI+MpCHQwpk11exjhF3etpPv47rFd6PiumeCsJxSPeFp6N2erRHayl0KhGq9toH9pMKdlu/fy2dWz1QLVe57ClEfDvz+mn89z467L2oLkQC3gi4szM1VwFPxR7TPx0dw4/nmNwQpivDVoPzYswtsqNxgH5BsXL9L3DFhRGrLyDvEkmFpFMhhCvLJQDB3KjPaxG5FQaA=
```

Comment:
One of those ENV variable was the CACHE URL clear, no more needed, the other one was an outdated GH TOKEN